cvat/apps/documentation/installation.md

```diff
@@ -307,7 +322,10 @@ nuctl deploy --project-name cvat \
     --platform local
 ```
 
-Note: see [deploy.sh](/serverless/deploy.sh) script for more examples.
+Note:
+- see [deploy.sh](/serverless/deploy.sh) script for more examples.
+- As an example, if you use `nuctl-1.4.18-linux-amd64` in the CVAT root directory,
+you need to change the `nuctl` in the above commands to `.nuctl-1.4.18-linux-amd64`.
```

Comment:
Probably a better approach is to rename nuclt-* to nuctl or make a soft link `ln nuctl-1.4.18-linux-amd64 nuctl`